bagisto/bagisto
# Fix registration notification issue for customer and admin

This commit resolves the issue where email notifications were not being sent to both customers and administrators upon successful registration when the email verification option was enabled. The fix ensures that both user types now receive the necessary registration notification emails.

Resolves: #8896

## Issue Reference
Resolves: #8896

## Description

**Customer.php Changes**
Added functionality to check the email verification option. If enabled, a new registration email is dispatched to the admin.

**RegistrationController.php Changes**
Upon successful customer email verification, a confirmation email for successful registration is sent to the customer.

**RegistrationNotification.php Changes**
Modified RegistrationNotification.php to handle email notification options. It now accepts a string constant ('admin' or 'customer') via the constructor. If the string constant is 'admin', an email is sent to the admin. If 'customer', an email is sent to the customer.

Created a Blade File (**notify-admin-about-registration.blade.php**) for sending admin emails and introduced new constants in **Resources/lang/en/app.php** for admin email content.

## How To Test This?
For testing purposes, follow these steps:

1. Enable the email verification option.
2. Register a new customer account.
3. Observe the notification dispatch under different configurations to ensure emails are sent appropriately based on the enabled verification options.

## Documentation
- [x] My pull request doesn't require updates on the documentation repository.
